.gitignore

```diff
@@ -2,3 +2,4 @@
 tmp
 .bundle
 **/vendor/bundle
+.idea
```

Comment:
if you put this into your global gitignore you never have to add it to any project